instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import CommentSection from "../CommentSection/CommentSection";
+
+const PostContainer = props => {
+  console.log(props);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    const newComment = {
+      id: 4,
+      username: "Team Yolo",
+      text: props.newComment
+    };
+    console.log(newComment, "test");
+    props.addNewComment(props.post.id, newComment);
+  };
+  return (
+    <div className="postContainer">
+      <div className="postHeader">
+        <img src={props.post.thumbnailUrl} alt="thumbnail" />
+        {props.post.username}
+      </div>
+      <div className="postBody">
+        <img className="mainPicture" src={props.post.imageUrl} alt="full" />{" "}
+        <div>
+          {props.post.likes} <p>{"likes"}</p>
```

Comment:
Great job writing up a comment section I can clearly follow the structure to. I would consider breaking this into sub components as well like a comment list, and a comment form or something.